### DISCUSSION TITLE
Restore GYB backup to a non-gmail imap account?

### DISCUSSION THREAD

**Discussion by maltokyo** [GAM-team/got-your-back#466] [Q&A]
I recently moved to mailcow, hosted on a VPS.  I have a backup of all gmail from GYB.  Would anyone know an easy way to get this into a standard non-gmail email account?  Can GYB do it?

**Comment by meksone** [comment]
See this a little late, but maybe my comment would be useful anyway :D
Since the backup is basically EML files, you can import using **Thunderbird** and **ImportExportToolsNG** extension.
But you can also export directly from your Google account using MBOX format, and then import again with ThunderBird/ImportExportToolsNG.
Alternatively, you can do a "live" backup, IMAP to IMAP, using [IMAPsync](https://imapsync.lamiral.info), just create an app password for Gmail if you use 2FA.

**Reply by maltokyo** [reply]
Thank you. Great. Will try this!

**Reply by meksone** [reply]
> Thank you. Great. Will try this!

Just some notes: restoring labels/folders using GYB backup on non-Gmail account isn't possibile; for restoring labels as folder, it's better to use direct IMAP to IMAP transfer. MBOX format from GoogleTakeout should retain label informations but I'm not sure (I've always used direct IMAP transfers using IMAPsync).
